modules/unsupported/process-raster/src/main/java/org/geotools/process/raster/ConvolveCoverageProcess.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2011-2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
These are new files, shouldn't the be just "(C) 2016, Open Source Geospatial Foundation (OSGeo)"
(also considering the last GSIP on the topic?)